client/blocks/inline-help/index.jsx

```diff
@@ -49,6 +54,15 @@ class InlineHelp extends Component {
 		}
 	}
 
+	componentDidUpdate( prevProps, prevState ) {
+		if ( prevState.showContactForm && prevProps.searchQuery !== this.props.searchQuery ) {
+			this.toggleContactForm();
+		}
+		if ( ! prevProps.isHappychatOpen && this.props.isHappychatOpen ) {
+			this.closeInlineHelp();
```

Comment:
Does this mean that when all operators go offline, if someone has the help popover open, it will close?

Reply:
`isHappychatOpen` is `true` when the Happychat "window" is visible. This line makes sure that we close Inline Help once we've successfully started a Happychat session (also: only if the "window" wasn't open before).

Reply:
(So, to answer your question — I don't believe so.)

Reply:
@lsinger is right — `isHappychatOpen` refers to whether the HC panel is visible or minimized. So this just means any time the Happychat panel opens up, inline help will go away.

There's a few quirks with this approach (if the HC panel is already open and the user opens inline help and fills out the Happychat form, the inline help popover won't go away). But those are rare edge cases that don't affect the primary customer experience, and are fixable with a refactor of the HelpContact form if this experiment is successful and we want to make this perfect for long-term maintenance.

Reply:
ohhhhh!  

I obviously didn't look too deeply & misinterpreted the "open" in `isHappychatOpen` to mean something like "available."

Thanks for clarifying.